azkaban-common/src/main/java/azkaban/jobExecutor/ProcessJob.java

```diff
@@ -174,10 +164,6 @@ public void run() throws Exception {
     generateProperties(propFiles[1]);
   }
 
-  private boolean determineExecuteAsUser(Props sysProps, Props jobProps) {
-    return sysProps.getBoolean(EXECUTE_AS_USER, false);
```

Comment:
I think it is always good to have an option. I believe it will be good to keep the config and just change the default to "true" so you can remove default value from all .properties file.